cypress/utils/components/cru-resource.po.ts

```diff
@@ -0,0 +1,139 @@
+import PagePo from '@/utils/components/page.po';
+import LabeledInputPo from '@/utils/components/labeled-input.po';
+import LabeledSelectPo from '@/utils/components/labeled-select.po';
+
+export default class CruResourcePo extends PagePo {
+  constructor({
+    type,
+    realType,
+  }: {
+    type: string,
+    realType?: string,
+  }) {
+    super(`/dashboard/c/local/harvester/${type}`);
+
+    this.type = type
+    this.realType = realType || type
+  }
+
+  private type = '';
+  private realType = '';
+
+  namespace() {
+    return new LabeledSelectPo('.labeled-select', `:contains("Namespace")`)
+  }
+
+  name() {
+    return new LabeledInputPo('.namespace-select > .labeled-input', `:contains("Name")`)
+  }
+
+  description() {
+    return new LabeledInputPo('.labeled-input', `:contains("Description")`)
+  }
+
+  public create(value: any) {
+    cy.visit(`/dashboard/c/local/harvester/${this.type}/create`)
+
+    this.setValue(value)
+
+    this.save()
+  }
+
+  public clone(id:string, value:any) {
+    cy.visit(`/dashboard/c/local/harvester/${this.type}/${id}?mode=clone`)
+
+    this.setValue(value)
+
+    this.save()
+  }
+  
+
+  public save() {
+    cy.intercept('POST', `/v1/harvester/${this.realType}s`).as('create');
+    cy.get('.cru-resource-footer').contains('Create').click()
```

Comment:
This cy.get request should be pointing at a class member not a CSS class. so something like
`private footer-button = '.cru-resource-footer';`

Reply:
Done